src/language/class-modifiers.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Class modifiers
+description: Modifier keywords for class declarations to control external library access.
+---
+
+Class modifiers control how a class or a mixin can be subtyped, both
```

Comment:
This is a tough one because it's typically outside libraries (so, not "you", probably someone else) attempting the subtyping. So you'd have to say "how _other libraries_ can subtype...". BUT, it's not exclusively about _other_ libraries, because it could also be from within the same library... Maybe:

> Class modifiers control _how a class or a mixin allows subtyping,_ both
[from within its own library](#abstract), and from outside of the library where
it's defined.

At this point I'm scared to change anything that hasn't undergone technical review, but I think this is safe... 😬